cmd/server.go

```diff
@@ -5,20 +5,16 @@ import (
 	"net/http"
 
 	"github.com/99designs/gqlgen/example/dataloader"
-	"github.com/99designs/gqlgen/handler"
 	"github.com/go-chi/chi"
-	"github.com/rafaelrubbioli/espenses/generated"
-	"github.com/rafaelrubbioli/espenses/resolver"
+	"github.com/rafaelrubbioli/espenses/pkg/graphql"
 )
 
 func main() {
 	router := chi.NewRouter()
 	router.Use(dataloader.LoaderMiddleware)
 
-	router.Handle("/", handler.Playground("Expenses", "/query"))
-	router.Handle("/query", handler.GraphQL(
-		generated.NewExecutableSchema(generated.Config{Resolvers: &resolver.Resolver{}}),
-	))
+	router.Handle("/", graphql.HandlePlayground())
+	router.Handle("/query", graphql.Handle())
```

Comment:
`func` is only useful if you want to inject something on the handle.
Instead you can simple use the GraphQL Playground Handle for now.
```suggestion
	router.Handle("/query", handler.Playground("Expenses", "/query"))
```